experimental/generated/beats/fields.ecs.yml

```diff
@@ -3652,6 +3652,52 @@
         system (nsfs). Unsigned int inum in include/linux/ns_common.h.
       example: 256383
       default_field: false
+    - name: risk.calculated_level
+      level: extended
```

Comment:
What's "level" here?

Reply:
It essentially just means either core ECS fields or the more use-case specific https://www.elastic.co/guide/en/ecs/current/ecs-guidelines.html#_ecs_field_levels